code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -259,7 +259,7 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 			var/u2_icon = U2.sprite_sheets && (dna.species.sprite_sheet_name in U2.sprite_sheets) ? U2.sprite_sheets[dna.species.sprite_sheet_name] : U2.icon
 			underwear_standing.Blend(new /icon(u2_icon, "us_[U2.icon_state]_s"), ICON_OVERLAY)
 
-	if(socks && dna.species.clothing_flags & HAS_SOCKS)
+	if(socks && dna.species.clothing_flags & HAS_SOCKS && (get_organ("l_leg") && get_organ("r_leg"))) // Check if the human has both legs before going on adding socks.
```

Comment:
this seems to be from the other PR?

Reply:
It is.... not intended, give me a second.